torch/hub.py

```diff
@@ -747,4 +750,4 @@ def load_state_dict_from_url(
 
     if _is_legacy_zip_format(cached_file):
         return _legacy_zip_load(cached_file, model_dir, map_location)
```

Comment:
`_legacy_zip_load()` will also make a call to `torch.load()`. I'm not sure this code path is still being hit nowadays, but to be on the safe side maybe we should also pass it `weight_only`?